.buildkite/hooks/pre-command

```diff
@@ -22,7 +21,8 @@ if [[ ("$BUILDKITE_PIPELINE_SLUG" == "golang-crossbuild" || "$BUILDKITE_PIPELINE
 fi
 
 if [[ "$BUILDKITE_PIPELINE_SLUG" == "golang-crossbuild" && "$BUILDKITE_STEP_KEY" == "release-post" ]]; then
-    export GITHUB_TOKEN_SECRET=$(retry 5 vault kv get -field token ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_USERNAME_SECRET=$(retry 5 vault kv get -field username ${GITHUB_TOKEN_VAULT_PATH})
-    export GITHUB_EMAIL_SECRET=$(retry 5 vault kv get -field email ${GITHUB_TOKEN_VAULT_PATH})
+    GITHUB_USERNAME_SECRET="elasticmachine"
+    export GITHUB_USERNAME_SECRET=$GITHUB_USERNAME_SECRET
+    export GITHUB_EMAIL_SECRET="elasticmachine@elastic.co"
+    export GITHUB_TOKEN_SECRET=$VAULT_GITHUB_TOKEN
```

Comment:
My review in https://github.com/elastic/golang-crossbuild/pull/421 also applied here, can you please rename the variables so the `_SECRET` suffix is not needed. cc @pazone

Reply:
Sorry that didn't answer in time. I created new PR because there was a problem to test it from fork.
I decide to not change names of variables because main issue was to change token to PAT and I didn't change names because it will follow with changes for all references to this variable in this repo. 
Also it is more secure to have SECRET in variable name which will prevent the variable from being shown in the pipeline.

Reply:
> Also it is more secure to have SECRET in variable name which will prevent the variable from being shown in the pipeline.

That's true if the values are not hardcoded plainly. Buildkite pipelines are not publicly available, as far as I know, but these changes are. 

If the `_SECRET` is not honoured, it might mislead the rationale for using it in the future, for maintenance reasons, I think it's better to create a follow-up and rename those env variables.